react/components/SearchQuery.js

```diff
@@ -169,6 +169,12 @@ const useCorrectSearchStateVariables = (
   return result
 }
 
+const skipSponsoredProducts = ({ sponsoredProductsBehavior, settings }) => {
+  const fetchSponsoredProductsConfig = settings?.fetchSponsoredProductsOnSearch
+
+  return !fetchSponsoredProductsConfig && sponsoredProductsBehavior === 'skip'
```

Comment:
It's still necessary to check for the `sponsoredProductsBehavior` value, since the account `exitocol` still won't abide to these changes. Once they do, we can remove this.

| Account state | Value of `fetchSponsoredProductsOnSearch` | Set the `sponsoredProductsBehavior`? | Boolean outcome | Will fetch ads? |
|--|--|--|--|--|
| Normal account | `false` | `false` |  `false` | No |
| VTEX Ads account | `true` | `false` | `true` | Yes |
| exitocol | `false` | `true` | `true` | Yes |

Reply:
@thalytafabrine Yes, of course!